File: week-04/day-4/src/Sum/Sum.java
package Sum;

import java.util.ArrayList;
import java.util.List;

public class Sum {
  List<Integer> numbers;

  public Sum(){
    this.numbers = new ArrayList<>();
  }

  public int sumNumbers(List<Integer> numbers){
    int sum = 0;
    for (int i = 0; i < numbers.size(); i++) {
      sum += numbers.get(i);
    }
    return sum;
  }
}

File: week-06/day-2/sum.c
#include <stdio.h>
#include <stdlib.h>

// - Write a function called `sum` that sum all the numbers
//   until the given parameter and returns with an integer

int sum(int number)
{
    int i;
    int sum;
    for (i = 1; i <= number; i++){
        sum += i;
    }
    return sum;
}

int main()
{
	int number;
	printf("Give me a number until you want to sum: ");
	scanf("%d", &number);
	printf("%d" , sum(number));

    return 0;
}
